backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/CoreData.java

```diff
@@ -2,8 +2,10 @@
 
 import de.bund.digitalservice.ris.caselaw.domain.court.Court;
 import de.bund.digitalservice.ris.caselaw.domain.lookuptable.documenttype.DocumentType;
+import jakarta.validation.constraints.Future;
 import jakarta.validation.constraints.PastOrPresent;
 import java.time.LocalDate;
+import java.time.LocalDateTime;
 import java.time.Year;
 import java.util.List;
 import lombok.Builder;
```

Comment:
Extract data to other class ("MetaData", "Verwaltungsdaten", "MetaInfo", "ReadOnlyInfo" or other....)

Reply:
@kiwikern Should we skip this? I would say that it doesn't have a high prio?

Reply:
Yup sure, I will create a jira ticket with open tasks and then we can re-evaluate in the end.